### ISSUE TITLE
Reported rules without their number of line

### ISSUE THREAD

**Issue by mimadrid** [hadolint/hadolint#201]
- [x] This is a bug report
- [ ] This is a feature request
- [X] I searched existing issues before opening this one

First of all, thanks for this awesome tool, keep going! :+1: 

I'm experiencing problems with the vim integration of https://github.com/w0rp/ale
And I think this report is related to https://github.com/w0rp/ale/issues/958 

Thanks a lot in advance and don't hesitate to ask for more information

### Expected behavior
/dev/stdin:1 DL3006 Always tag the version of an image explicitly
/dev/stdin:2 DL4000 MAINTAINER is deprecated*
/dev/stdin:3 SC2154 node_verion is referenced but not assigned (did you mean 'node_version'?).
/dev/stdin:3 DL3009 Delete the apt-get lists after installing something
/dev/stdin:3 DL3015 Avoid additional packages by specifying `--no-install-recommends`
/dev/stdin:6 DL3003 Use WORKDIR to switch to a directory
/dev/stdin:6 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`
/dev/stdin:9 DL3011 Valid UNIX ports range from 0 to 65535
/dev/stdin:12 DL4004 Multiple `ENTRYPOINT` instructions found. If you list more than one `ENTRYPOINT` then only the last `ENTRYPOINT` will take effect

### Actual behavior

/dev/stdin DL4000 MAINTAINER is deprecated
/dev/stdin DL4004 Multiple `ENTRYPOINT` instructions found. If you list more than one `ENTRYPOINT` then only the last `ENTRYPOINT` will take effect
/dev/stdin:1 DL3006 Always tag the version of an image explicitly
/dev/stdin:3 SC2154 node_verion is referenced but not assigned (did you mean 'node_version'?).
/dev/stdin:3 DL3009 Delete the apt-get lists after installing something
/dev/stdin:3 DL3015 Avoid additional packages by specifying `--no-install-recommends`
/dev/stdin:6 DL3003 Use WORKDIR to switch to a directory
/dev/stdin:6 DL3016 Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`
/dev/stdin:9 DL3011 Valid UNIX ports range from 0 to 65535

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.6.5-0-g933c2e2
```

**Dockerfile** (if relevant)

```Dockerfile
FROM debian
MAINTAINER test
RUN export node_version="0.10" \
 && apt-get update && apt-get -y install nodejs="$node_verion"
COPY package.json usr/src/app
RUN cd /usr/src/app \
 && npm install node-static

EXPOSE 80000
ENTRYPOINT ["test1"]
ENTRYPOINT ["test2"]
CMD ["npm", "start"]

```

**Additional environment details (OS, stack version, etc.)**

**Comment by mimadrid** [comment]
I'm using the last ALE version and it's working fine except that the regular expression of ALE only parses rules with number of line associated so the next two rules aren't shown (lines 2 and 11)
> /dev/stdin DL4000 MAINTAINER is deprecated
> /dev/stdin DL4004 Multiple ENTRYPOINT instructions found. If you list more than one ENTRYPOINT 

I hope to be useful

![hadolint_arrows](https://user-images.githubusercontent.com/3269878/40427281-87e7b3ce-5e9d-11e8-9750-2432d20ed179.png)

**Comment by lorenzo** [comment]
are you sure you are running the latest hadolint as well? That does not look like the output of hasolint, as it always outputs the line number

**Comment by lorenzo** [comment]
Can you include your dockerfile so I can try locally?

**Comment by lorenzo** [comment]
@mimadrid I can reproduce your issue, will get it fixed shortly

**Comment by mimadrid** [comment]
@lorenzo, thanks a lot, you're awesome! :+1: 
Don't hesitate to ask for testing :)

**Comment by mimadrid** [comment]
Thanks a lot! it looks great! :100:
